### PR TITLE
Alternative communicator and cane sprites

### PR DESCRIPTION
**Pull request by Greenjoe12345**
This adds several new items:
two new styles of communicator, one based on a flip phone and one based on a old nokia kinda design.
a folding white cane, and a telescopic walking cane.
sprites are from aurora and tg.
image

### CODE REVIEW

**Comment by Cerebulon** [comment]
I think it was generally be a good idea to just name these PRs something more descriptive like "Alternative communicator and cane sprites" because "2022 end item additions" sounds like some massive overhaul and this is essentially just a couple of sprites.

**Comment by Greenjoe12345** [comment]
Fair enough

File: code/game/objects/items/devices/communicator/communicator.dm [hunk 1]
```diff
@@ -387,3 +386,43 @@ var/global/list/obj/item/communicator/all_communicators = list()
...
 	icon_state = initial(icon_state)
+
+/obj/item/communicator/flip
+	desc = "The S-12 Flip, An affordable communicator with a clamshell design."
+	icon_state = "commflip"
+	note = "Thank you for choosing the S-12 Flip, this is your notepad!" //Current note in the notepad function
+
+/obj/item/communicator/flip/update_icon()
+	if(video_source)
+		icon_state = "commflip-video"
```
Comment (Atermonera) [inline code/game/objects/items/devices/communicator/communicator.dm:397]: Use `"[initial(icon_state)-video"` in the parent type for these?

File: code/modules/client/preference_setup/loadout/loadout_general.dm [hunk 2]
```diff
@@ -2,6 +2,10 @@
 	display_name = "cane"
 	path = /obj/item/cane
 
+/datum/gear/cane/telescopic
```
Comment (Atermonera) [inline code/modules/client/preference_setup/loadout/loadout_general.dm:5]: Gear-selection datum for all the subtypes of cane?

**Review by Atermonera — CHANGES_REQUESTED** [review]
Sorry for my tardiness here, last week was pretty busy.

**Comment by Greenjoe12345** [comment]
I've done those adjustments.

**Review by Atermonera — APPROVED** [review]

**Pull request merged**